### ISSUE TITLE
libbrewscheme self compile

### ISSUE THREAD

**Issue by wbuchanan** [wbuchanan/brewscheme#20]
Add code that will check for existing libbrewscheme.mlib file and compile if it does not exist to prevent users's from needing to compile the Mata library themselves.

**Comment by randomgambit** [comment]
good idea. I struggled quite a bit to understand how to install your package actually !!! ;-)

**Comment by wbuchanan** [comment]
Right now the difficulty is accessing some of the file system properties to check when the file was created/modified.  I may need to try putting something together in Java but will need to have a mechanism that allows the ado to be triggered if the existing file was created before a current version of the file.  I figured the distribution date would make things a bit easier since that is the same mechanism used by the update command.  If you know of existing ways to access file properties let me know and I can check those out as well.  It may take a bit longer to resolve this one than I anticipated originally.
